lib/osf-components/addon/components/storage-provider-manager/provider-mapper/template.hbs

```diff
@@ -1,3 +1,14 @@
 {{yield (hash
-    osfstorage=(component 'storage-provider-manager/osf-storage-manager')
+    bitbucket=(component 'storage-provider-manager/storage-manager')
+    box=(component 'storage-provider-manager/storage-manager')
+    dataverse=(component 'storage-provider-manager/storage-manager')
+    dropbox=(component 'storage-provider-manager/storage-manager')
+    figshare=(component 'storage-provider-manager/storage-manager')
+    github=(component 'storage-provider-manager/storage-manager')
+    gitlab=(component 'storage-provider-manager/storage-manager')
+    googledrive=(component 'storage-provider-manager/storage-manager')
+    onedrive=(component 'storage-provider-manager/storage-manager')
+    osfstorage=(component 'storage-provider-manager/storage-manager')
+    owncloud=(component 'storage-provider-manager/storage-manager')
+    s3=(component 'storage-provider-manager/storage-manager')
```

Comment:
If all of them returns the same `StorageManager` component, then does it even make sense to have a mapper component?
I guess we can anticipate a future provider whose behavior is so significantly different that requires a customized `StorageManager`. But at this moment this seems redundant.

Reply:
I thought about removing the mapper, but was also concerned about future providers. I figured rather than throw out the capability, it would make sense to just not take advantage of it for now. Maybe after phase 2, once we're sure we don't need it right now, we could pull it out.